src/main/java/competition/subsystems/armlifting/LiftingSubsystem.java

```diff
@@ -0,0 +1,53 @@
+package competition.subsystems.armlifting;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+
+import xbot.common.command.BaseSubsystem;
+import xbot.common.injection.wpi_factories.CommonLibFactory;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.PropertyFactory;
+ 
+@Singleton
+public class LiftingSubsystem extends BaseSubsystem {
```

Comment:
This should be renamed something like `CollectorArmSubsystem` - with just Lifting, I think about lifting the whole robot, not just the arm.

Reply:
done